USAGE.md

```diff
@@ -0,0 +1,130 @@
+# How to use TiKV
+
+## Getting started
+
+### System requirement
+
++ Linux or Mac OSX.
++ Rust, nightly version is required.
++ Go, 1.5+ is required.
+
+### Installing TiKV
+
++ RocksDB
+
+    ```sh
+    curl -L https://github.com/facebook/rocksdb/archive/rocksdb-4.3.1.tar.gz -o rocksdb.tar.gz
+    tar xf rocksdb.tar.gz 
+    cd rocksdb-rocksdb-4.3.1 
+    make shared_lib 
+    make install-shared
+    ```
+
++ Etcd
+
+    It is required to use the latest etcd version which supports gRPC from the master branch.
```

Comment:
'the latest version (master branch or 3.0+) of etcd is required for its gRPC support.'